src/components/traces/details/traceDetails.less

```diff
@@ -17,36 +17,72 @@
 
 @import (reference) '../../../app';
 
-.span-label {
-    font-weight: 700;
-    font-size: @font-size-base;
-    cursor: default;
-    fill: #6B7693
+.time-pointer {
+    fill: @gray-dark;
 }
 
-.span-service-label {
-    font-size: @font-size-base;
-    font-weight: 600;
-    fill: #FFFFFF;
-    cursor: default;
+.time-pointer-line {
+    stroke: @gray-light;
+    stroke-width: 1;
+    stroke-opacity: 0.5;
 }
 
-.span-bar {
-    rx: 3.5;
-    ry: 3.5;
-    fill: #28a228
+.timeline-transition {
```

Comment:
Is this supposed to be tied to something? I don't see another timeline-transition anywhere.